charts/apisix-dashboard/values.yaml

```diff
@@ -89,6 +89,16 @@ config:
       username: ~
       # -- Specifies etcd basic auth password if enable etcd auth
       password: ~
+
+      # -- Specifies a secret to be mounted on /etc/etcd for mtls usage
+      existingSecret: ""
```

Comment:
I think this name is ambiguous. It would be better to add mtls to it.

Reply:
done